Formula/gusto-lima.rb

```diff
@@ -0,0 +1,70 @@
+class GustoLima < Formula
+  desc "Gusto's opinionated colima profile"
+  homepage "https://github.com/abiosoft/colima/blob/main/README.md"
+  url "https://github.com/abiosoft/colima.git",
+      tag:      "v0.1.0"
+  license "MIT"
+  head "https://github.com/abiosoft/colima.git", branch: "main"
+
+  depends_on "colima"
+  depends_on "docker"
+  depends_on "docker-buildx"
+  depends_on "docker-compose"
+  depends_on "docker-credential-helper"
+  depends_on "docker-credential-helper-ecr"
+
+  def install
+
+    bin.install_symlink "/opt/homebrew/opt/colima/bin/colima" => "gusto-lima"
```

Comment:
This would break on intel right? I think there is a brew helper for this

Reply:
updated in latest commit